File: app/src/main/java/com/decode/tumblr/model/Trail.java
package com.decode.tumblr.model;

import com.google.gson.annotations.SerializedName;

public class Trail {

    @SerializedName("blog")
    public Blog_ blog;
    @SerializedName("post")
    public Post_ post;
    @SerializedName("content_raw")
    public String contentRaw;
    @SerializedName("content")
    public String content;
    @SerializedName("is_root_item")
    public Boolean isRootItem;

}

File: app/src/main/java/com/decode/tumblr/data/model/Trail.kt
package com.decode.tumblr.data.model

import com.google.gson.annotations.SerializedName

data class Trail(

        @SerializedName("blog")
        var blog: Blog_? = null,

        @SerializedName("post")
        var post: Post_? = null,

        @SerializedName("content_raw")
        var contentRaw: String? = null,

        @SerializedName("content")
        var content: String? = null,

        @SerializedName("is_root_item")
        var isRootItem: Boolean? = null

)
